packages/ckeditor5-dev-env/lib/release-tools/tasks/generatechangelogforsinglepackage.js

```diff
@@ -25,9 +25,11 @@ const transformCommitFunction = require( '../utils/transform-commit/transformcom
  * should be generated.
  *
  * @param {String|null} [newVersion=null] A version for which changelog will be generated.
+ * @param {Object} [options={}] Additional options.
```

Comment:
`newVersion` should be a part of `options`. It's completely optional and not significant enough to be kept as a separate param.

Reply:
Fixed in 5684978.